permissions/plugin-wallarm-fast.yml

```
@@ -0,0 +1,7 @@
+---
+name: "wallarm-fast"
+github: "jenkinsci/wallarm-fast-plugin"
+paths:
+- "org/jenkins-ci/plugins/wallarm-fast"
```

The path should use your groupId from your pom.xml.

```suggestion
- "io/jenkins/plugins/wallarm-fast"
```